### ISSUE TITLE
tensorflow——线下分享预热

### ISSUE THREAD

**Issue by ocAreTheBestLanguage** [100steps/Blogs#69]
一、什么是TensorFlow？
二、MNIST数据集（手写数字识别）
	①：sigmoid和softmax
	③：cross_entropy （交叉熵）
	④：gradientdescent（梯度下降）
三、CNN卷积神经网络（Convolutional Neural Network）
	①：什么是卷积神经网络
	②：参数减少和权重共享
	③：strides、patch、padding
	③：最大池化

**Comment by ocAreTheBestLanguage** [comment]
https://www.tensorflow.org/
（google的tensorflow官网）
https://hit-scir.gitbooks.io/neural-networks-and-deep-learning-zh_cn/content/chap3/c3s1.html
（不错的gitbook）
[https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist.py](url)
一个会讲到的入门例子
[https://github.com/MorvanZhou/tutorials/blob/master/tensorflowTUT/tf18_CNN3/full_code.py](url)
一个会讲到的进阶例子
